src/dom.js

```diff
@@ -544,3 +544,18 @@ export function escapeCssSelectorIdent(win, ident) {
   // Polyfill.
   return cssEscape(ident);
 }
+
+
+/**
+ * Returns a frame element if available.
+ * @param {!Window} win
+ * @return {?HTMLIFrameElement}
+ */
+export function getFrameElement(win) {
+  try {
+    return win.frameElement;
+  } catch (e) {
```

Comment:
I thought you were just concerned about failures on some random browser. Originally there wasn't a `try/catch`. Really by the time I check `frameElement` in all cases, I'm already 100% sure I'm working with a friendly frame. I can either kill this method, or do defense in depth here and check that this iframe been marked as friendly. WDYT?